code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -222,25 +229,23 @@ GLOBAL_LIST_EMPTY(ts_spiderling_list)
 /mob/living/simple_animal/hostile/poison/terror_spider/examine(mob/user)
 	. = ..()
 	var/list/msgs = list()
-	if(stat == DEAD)
-		msgs += "<span class='notice'>It appears to be dead.</span>\n"
-	else
+	if(stat != DEAD)
 		if(key)
-			msgs += "<span class='warning'>Its eyes regard you with a curious intelligence.</span>"
+			msgs += "<span class='warning'>[p_they(TRUE)] regards [p_their()] surroundings with a curious intelligence.</span>"
```

Comment:
can just do `. += message`.

Same goes for the others as well. Don't have to join manually here